tests/test_cli.py

```diff
@@ -1,24 +1,55 @@
 import os
+import pytest
 
 from click.testing import CliRunner
 
 from cookiecutter.cli import main
+from cookiecutter import utils
 
 runner = CliRunner()
 
 
+@pytest.fixture
+def remove_fake_project_dir(request):
+    """
+    Remove the fake project directory created during the tests.
+    """
+    def fin_remove_fake_project_dir():
+        if os.path.isdir('fake-project'):
+            utils.rmtree('fake-project')
+    request.addfinalizer(fin_remove_fake_project_dir)
+
+
+@pytest.fixture
+def make_fake_project_dir(request):
+    """
+    Create the fake project directory created during the tests.
+    """
+    if not os.path.isdir('fake-project'):
+        os.makedirs('fake-project')
+
+
 def test_cli_version():
     result = runner.invoke(main, ['-V'])
     assert result.exit_code == 0
     assert result.output.startswith('Cookiecutter')
 
 
+@pytest.mark.usefixtures('make_fake_project_dir', 'remove_fake_project_dir')
+def test_cli_error_on_existing_output_directory():
+    result = runner.invoke(main, ['tests/fake-repo-pre/', '--no-input'])
+    assert result.exit_code != 0
+    assert result.output.startswith('Error:')
```

Comment:
Yes that makes sense since there is a chance that the error message could be for some other error and the test does not verify it at the moment. Will fix this now.